analyzers/src/SonarAnalyzer.Common/Helpers/SonarLintXmlReader.cs

```diff
@@ -32,49 +32,55 @@ public class SonarLintXmlReader
 
     private readonly SonarLintXml sonarLintXml;
 
-    private bool? ignoreHeaderCommentsCS;
-    private bool? ignoreHeaderCommentsVB;
+    public string[] Exclusions { get => Exclusions; init => ReadCommaSeparatedArray(ReadSettingsProperty("sonar.exclusions")); }
```

Comment:
I don't like that I'm obliged to define the `get` - but for now, it's either this or put back everything in the constructor and leave the Properties as {get; init;} (`ReadSettingsProperty` is not static).

Reply:
Outdated